src/mscorlib/src/System/Environment.cs

```diff
@@ -1239,9 +1249,9 @@ public static IDictionary GetEnvironmentVariables()
             }
         }
 
-        #if FEATURE_CORECLR
+#if FEATURE_CORECLR
```

Comment:
This is the right formatting I believe, VS auto formatted when updating the #if preprocessor statements.